src/authenticators/unix_peer_credentials_authenticator/mod.rs

```diff
@@ -247,7 +249,8 @@ mod test {
             .authenticate(&req_auth, conn_metadata)
             .expect("Failed to authenticate");
 
-        assert_eq!(application.identity.name, get_current_uid().to_string());
+        let current_uid: uid_t = unsafe { getuid() };
```

Comment:
Can we reuse the `current_uid` variable used before?

Reply:
Done